osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -89,6 +90,14 @@ private void load(BeatmapListingOverlay beatmapListing, SettingsOverlay settings
                     }
                 });
             }
+            else if (host.CanSuspendToBackground)
+            {
+                AddInternal(exitConfirmOverlay = new NoHoldExitConfirmOverlay
+                {
+                    // treat as if the UIHoldActivationDelay is always 0. see NoHoldExitConfirmOverlay xmldoc for more info.
+                    Action = this.Exit
+                });
```

Comment:
I would've simplified this into making `MainMenu` inherit `IKeyBindingHandler<GlobalAction>` and called the suspend method in there, instead of introducting a "no-hold confirm overlay". But sure I suppose?